Sources/StreamChat/Controllers/CurrentUserController/CurrentUserController.swift

```diff
@@ -193,30 +193,26 @@ public extension CurrentChatUserController {
     }
     
     /// Registers a device to the current user.
-    /// `setUser` must be called before calling this.
+    /// `connectUser` must be called before calling this.
     /// - Parameters:
     ///   - token: Device token, obtained via `didRegisterForRemoteNotificationsWithDeviceToken` function in `AppDelegate`.
+    ///   - asCurrentDevice: Specifies if the device should be saved as `currentDevice` for the user. Defaults to `true`.
     ///   - completion: Called when device is successfully registered, or with error.
-    func addDevice(token: Data, completion: ((Error?) -> Void)? = nil) {
+    func addDevice(token: Data, asCurrentDevice: Bool = true, completion: ((Error?) -> Void)? = nil) {
```

Comment:
@b-onc whats the usecase for `asCurrentDevice: false`?

Reply:
As a provider, I keep the user's device tokens in my own backend, and when they install an updated client (which includes StreamChat SDK) to one of their devices I want to enable push notifications for all of their devices. So I add all of their devices from a single client